libultraship/libultraship/SohConsole.cpp

```diff
@@ -317,7 +319,7 @@ int Console::CallbackStub(ImGuiInputTextCallbackData* data) {
 	return 0;
 }
 
-void Console::Append(const std::string& channel, Priority priority, const char* fmt, ...) IM_FMTARGS(4) {
+void Console::Append(const std::string& channel, Priority priority, const char* fmt, ...) {
```

Comment:
Is this something that will still work in msvc? Should it be ifdefed for linux?

Reply:
This attribute doesn't make any sense in the function definition in the first place if there's a forward declaration in the header.